bibliography.bib

```diff
@@ -7136,9 +7136,508 @@ @Article {ProHsiKum2017IEEETR
+    doi          = {10.1145/3092815},
+    note         = {},
+    annote       = {},
+}
+
+@Article {SilParChr2017RSOS,
+    title        = {Evolutionary online behaviour learning and adaptation in real robots},
+    author       = Silva_F # and # Correia_L # and # Christensen_AL,
+    journal      = RSOS,
+    series       = {},
+    volume       = {4},
+    number       = {7},
+    year         = {2017},
+    pages        = {160938},
+    doi          = {10.1098/rsos.160938},
+    note         = {},
+    annote       = {},
+}
+
+@Comment{The journal issue appeared in January 2022, but the publisher suggests 2021 as publication year.}
+@Article {BreFon2021PTRSLSB,
+    title        = {Social learning in swarm robotics},
+    author       = Bredeche_N # and # Fontbonne_N,
+    journal      = PTRSLSB,
+    series       = {},
+    volume       = {377},
+    number       = {1843},
+    year         = {2021},
+    pages        = {20200309},
+    doi          = {10.1098/rstb.2020.0309},
+    note         = {},
+    annote       = {},
+}
+
+@InProceedings {HeiRanEib2015ssci,
+    title        = {Evolution, individual learning, and social learning in a swarm of real robots},
+    author       = Heinerman_J # and # Rango_M # and # Eiben_AE,
+    booktitle    = ssci_2015,
+    editor       = {},
+    series       = {},
+    volume       = {},
+    number       = {},
+    year         = {2015},
+    pages        = {1055--1062},
+    publisher    = pub_IEEECS,
+    address      = LosAlamitos_CA_USA,
+    doi          = {10.1109/SSCI.2015.152},
+    note         = {},
+    annote       = {},
+}
+
+@InProceedings {DonMouBrePad2011evoderob,
+    title        = {Evolutionary robotics: exploring new horizons},
+    author       = Doncieux_S # and # Mouret_JB # and # Bredeche_N # and # Padois_V,
+    booktitle    = evoderob_2009,
+    editor       = Doncieux_S # and # Bredeche_N # and # Mouret_JB,
+    series       = srs_SCI,
+    volume       = {341},
+    number       = {},
+    year         = {2011},
+    pages        = {3--25},
+    publisher    = pub_SP,
+    address      = Berlin_DEU,
+    doi          = {10.1007/978-3-642-18272-3_1},
+    note         = {},
+    annote       = {},
+}
+
+@InCollection {TriTucAmpDor2014horizons,
+    title        = {Evolutionary swarm robotics: a theoretical and methodological itinerary from individual neuro-controllers to collective behaviours},
+    author       = Trianni_V # and # Tuci_E # and # Ampatzis_C # and # Dorigo_M,
+    booktitle    = {The Horizons of Evolutionary Robotics},
+    editor       = Vargas_PA # and # DiPaolo_EA # and # Harvey_I # and # Husbands_P,
+    series       = {},
+    volume       = {},
+    number       = {},
+    year         = {2014},
+    pages        = {153--178},
+    publisher    = pub_MITP,
+    address      = Boston_MA_USA,
+    doi          = {10.7551/mitpress/8493.003.0008},
+    note         = {},
+    annote       = {},
+}
+
+@InProceedings {GhaKucGarBir2023icra,
```

Comment:
Maybe add a comment here (besides the note) so we don't forget to update the entry once a decision has been made?

Reply:
Added a comment